web/src/compose.js

```diff
@@ -373,6 +372,12 @@ function schedule_message_to_custom_date() {
     } else {
         req_type = message_type;
     }
+    //valueof() gives us the timestamp in milliseconds so dividing by 1000 gives us seconds
+    const now = new Date().valueOf() / 1000;
+    const minimum_schedule_msg_delay = scheduled_messages.MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS;
+    const time_difference = Math.abs(scheduled_delivery_timestamp - now);
+    const add_notice = time_difference < minimum_schedule_msg_delay ? true : false;
+    const minimum_schedule_msg_delay_in_minutes = minimum_schedule_msg_delay / 60;
```

Comment:
We can directly use `scheduled_messages.MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS / 60` in the `render_success_message_...` function.